tables/he.utb

```diff
@@ -0,0 +1,376 @@
+# liblouis: Hebrew table which supports multi-language texts and it is used in Israel.
+#
+# Copyright (C) 2020, BAUM Engineering <liblouis@baum.ro>
+#
+# This file is part of liblouis.
+#
+# liblouis is free software: you can redistribute it and/or modify it
+# under the terms of the GNU Lesser General Public License as
+# published by the Free Software Foundation, either version 2.1 of the
+# License, or (at your option) any later version.
+#
+# liblouis is distributed in the hope that it will be useful, but
+# WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the GNU
+# Lesser General Public License for more details.
+#
+# You should have received a copy of the GNU Lesser General Public
+# License along with liblouis. If not, see
+# <http://www.gnu.org/licenses/>.
+
+#-index-name: Hebrew, uncontracted
+#-display-name: Hebrew braille
+
+#+locale: he
+#+type: literary
+#+dots: 6
+#+contraction: no
+#+grade: 1
+
+#-license: LGPLv2.1
+#-author-name: BAUM Engineering
+#-author-email: liblouis@baum.ro
+
+# Braille indicators
+capsletter 6
+begcapsword 6-6
+endcapsword 6-3
+
+# literary forms of the decimal digits
+include litdigits6Dots.uti
+
+# display/passthrough for unicode braille
+include braille-patterns.cti
+
+# punctuation
+prepunc ( 2356
+postpunc ) 2356
+prepunc [ 6-2356
+postpunc ] 2356-3
+prepunc { 12346
+postpunc } 13456
+prepunc " 236
+postpunc " 356
+prepunc ` 6-236
+prepunc ' 6-236
+postpunc ' 356-3
+begnum # 456-1456 print number sign before number
+punctuation , 2
+decpoint . 46
+postpunc . 256
+punctuation ; 23
+punctuation : 25
+punctuation ! 235
+math < 25-246
+math > 135-25
+punctuation ? 26
+endnum st 34
+endnum nd 1345-145
+endnum rd 1235-145
+noback endnum 's 3-234
```

Comment:
Sorry, another thing that is bugging me: two lines below there is exactly the same rule without `noback`. That doesn't make sense.

Reply:
I agree and I have removed the duplicate noback one.